includes/functions.php

```diff
@@ -580,7 +580,10 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         if (host_exists($host) === false) {
             $device_id = dbInsert($device, 'devices');
             if ($device_id) {
-                return($device_id);
+                if ($config['oxidized']['enabled'] === TRUE && $config['oxidized']['reload_nodes'] === TRUE && isset($config['oxidized']['url'])) {
+                    $oxidized_reload_response = oxidized_reload_nodes();
+                }
+                return($device_id . $oxidized_reload_response);
```

Comment:
You can't return more than device_id here as it's used when calling the function.